shared/prometheus/service_test.go

```diff
@@ -94,7 +94,7 @@ func TestHealthz(t *testing.T) {
 	rr = httptest.NewRecorder()
 	handler.ServeHTTP(rr, req)
 
-	if status := rr.Code; status != http.StatusOK {
+	if status := rr.Code; status != http.StatusServiceUnavailable {
 		t.Errorf("expected OK status but got %v", rr.Code)
```

Comment:
```suggestion
		t.Errorf("expected StatusServiceUnavailable status but got %v", rr.Code)
```
Err message needs to be changed?

Reply:
Yep, thanks!